BUILD.md

```diff
@@ -20,11 +20,11 @@ There are also a few forks of existing packages under the "fork-*" name.
 
 ## Required dependencies
 
-- Install node 10+ - https://nodejs.org/en/
+- Install node 10+ (but not node 15) - https://nodejs.org/en/
```

Comment:
If the app cannot be built under Node 15, it's more a bug that should be fixed (and that I'll fix when I upgrade) rather than something to add to the doc, so please could you remove this part?

Reply:
Oh, okay. I mean, it's still perhaps worth mentioning in some way or another if Joplin hasn't been tested yet under Node 15. I specifically ran into the issues on Mac, for whatever that's worth.